connection/cluster/ClusterDatabaseManager.java

```diff
@@ -45,7 +45,7 @@
 
 public class ClusterDatabaseManager implements DatabaseManager.Cluster {
 
-    private static final Logger LOG = LoggerFactory.getLogger(FailsafeTask.class);
+    private static final Logger LOG = LoggerFactory.getLogger(ClusterClient.FailsafeTask.class);
```

Comment:
Hmm looks like this was always wrong
```suggestion
    private static final Logger LOG = LoggerFactory.getLogger(ClusterDatabaseManager.class);
```

Reply:
Oops, fixed in https://github.com/vaticle/typedb-client-java/pull/346/commits/e4a78991a2e7d340210206d22f8c8770d23c2ac1